entity/Comment.php

```diff
@@ -0,0 +1,33 @@
+﻿<?php
+
+class Comment
+{
+    private User $user;
+    private string $comment;
+    private DateTime $createdAt;
+
+    public function __construct(User $user, string $comment)
+    {
+        $this->user = $user;
+        $this->comment = $comment;
+        $this->createdAt = $this->randomDate(new DateTime('2022-01-01'), new DateTime());
+    }
+
+    private function randomDate($start_date, $end_date)
+    {
+        $min = strtotime($start_date);
```

Comment:
PHP Fatal error:  Uncaught TypeError: strtotime(): Argument #1 ($datetime) must be of type string, DateTime given